bin/zkCleanup.sh

```diff
@@ -31,27 +31,44 @@
 # use POSIX interface, symlink is followed automatically
 ZOOBIN="${BASH_SOURCE-$0}"
 ZOOBIN="$(dirname "$ZOOBIN")"
-ZOOBINDIR="$(cd "$ZOOBIN" && pwd)"
+ZOOBINDIR="$(cd "$ZOOBIN" || exit && pwd)"
```

Comment:
Will this exist the entire script, or just the subshell that the `cd` command is running in?
It might be better to do something like:


```suggestion
ZOOBINDIR="$(cd "$ZOOBIN" && pwd)" || exit
```

This is a little more clear what is going to happen, especially when it comes to the order of operations between `||` and `&&`, which are not immediately obvious to non-bash experts.